homeassistant/components/bthome/sensor.py

```diff
@@ -150,7 +150,7 @@
     # Used for moisture sensor
     (None, Units.PERCENTAGE,): SensorEntityDescription(
         key=f"{DeviceClass.MOISTURE}_{Units.PERCENTAGE}",
-        device_class=None,
+        device_class=SensorDeviceClass.MOISTURE,
```

Comment:
We need to move this change to a new PR because moisture is not part of the beta release.

```suggestion
        device_class=None,
```